examples/bpk-component-calendar/examples-components.js

```diff
@@ -289,12 +289,10 @@ class CalendarContainer extends Component {
 }
 
 CalendarContainer.propTypes = {
-  selectTodaysDate: PropTypes.bool,
```

Comment:
I deleted this from everywhere in the code because it doesn't seem to be an actual prop except for the example components, but even then it's never actually used!